src/spectroscopy.jl

```diff
@@ -1 +1,57 @@
 # This file should contain all spectroscopy specifc stuff like correlators, fits, effective masses, ...
+function effectivemass(c)
+    N = length(c)
+    m = similar(c)
+    for i in 1:N
+        r = c[i]/c[mod1(i+1,N)]
+        m[i] = r > 0 ? abs(log(r)) : NaN
```

Comment:
Not sure about the `abs()` function here.
Personally I prefer to keep the minus sign explicitely. Though I am not sure what is actually the ￼￼￼correct way according to the community.

Reply:
I don't think that there is a correct way of doing this, but all books seem to use the version without 'abs()'. Of course this does not matter if you only consider the range  up to $T/2$ so I am fine with dropping the  'abs()'.

Reply:
Yeah I also think without `abs()` is a bit cleaner (and probably minimally faster 😉).
Especially without this you can more easily differ between a `sinh()`- or `cosh()`-behaviour wether the interpolators are sink or source ones.